### PR TITLE
Add new guides to developer site - Run local network , Run Move programs locally.

### PR DESCRIPTION
**Pull request by archjayas**
## Motivation

* Provide a guide to run a local network of nodes using libra_swarm. The local network can be used by developers to test and debug code changes. Move programs can be compiled, published, and executed on this network.
* Provide a guide to compile, publish, and execute Move programs locally.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/website/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes.)

## Test Plan
Review the content on the two new pages "Run a Local Network" and "Run Move Porgrams Locally"
Review the usage of the CLI 'dev' command on the Libra CLI page.

Follow the instructions in the tutorial  "Run a local network" to:
* Run libra_swarm and CLI client in the same process.
* Run libra_swarm and CLI in  separate processes.

Follow the instructions in the tutorial  "Run Move programs locally" to:
- Compile and publish a Move module.
- Compile and execute a Move transaction script.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: docs/run-move-locally.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+libra% query balance 0
+Balance is: 76.000000
+```
+
+### Compile Move Module
+
+To compile `my_module.mvir`, use the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+```
+libra% dev compile 0 <path to my_module.mvir> module
```
Comment (kphfb) [inline docs/run-move-locally.md]: Should we say what the "0" means?

File: docs/run-move-locally.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+To create, compile, and publish Move modules to an account on the local blockchain, follow the instructions in [compile and publish Move modules](#compile-and-publish-custom-move-module). To compile and execute a Move transaction script, follow the instructions in [compile and execute transaction scripts](#compile-and-execute-custom-transaction-script).
```
Comment (sblackshear) [inline docs/run-move-locally.md]: Looks like the anchor links here are broken--this should fix it

```suggestion
To create, compile, and publish Move modules to an account on the local blockchain, follow the instructions in [compile and publish Move modules](#compile-and-publish-custom-move-modules). To compile and execute a Move transaction script, follow the instructions in [compile and execute transaction scripts](#compile-and-execute-custom-transaction-scripts).
```

File: docs/run-move-locally.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+```
+Successfully compiled a program at /var/folders/tq/8gxrrmhx16376zxd5r4h9hhn_x1zq3/T/.tmpDZhL21
+```
+
+If you would like to save the bytecode file at a specific path, and not at a temporary path, you can specify it as an argument to the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+### Execute Transaction Script
+
+To execute your script, use the [dev execute](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command on the bytecode output from [Compile the Transaction Script](#compile-the-transaction-script) step above.
```
Comment (sblackshear) [inline docs/run-move-locally.md]: Similar here: 
```suggestion
To execute your script, use the [dev execute](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command on the bytecode output from [Compile Transaction Script](#compile-transaction-script) step above.
```

File: docs/run-move-locally.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+```
+
+### Compile Move Module
+
+To compile `my_module.mvir`, use the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+```
+libra% dev compile 0 <path to my_module.mvir> module
+```
+The parameter `module` indicates that you are compiling a Move module. If you were compiling a transaction script, you would use `script` instead.
```
Comment (sblackshear) [inline docs/run-move-locally.md]: To address kph's feedback:

```suggestion
The 0 parameter is the index of the account that the module will be published under. The parameter `module` indicates that you are compiling a Move module. If you were compiling a transaction script, you would use `script` instead.
```

File: docs/run-move-locally.md (hunk 1 above)
Comment (archjayas) [reply docs/run-move-locally.md]: Added the description for 0 and updated the PR.

File: docs/run-move-locally.md (hunk 2 above)
Comment (archjayas) [reply docs/run-move-locally.md]: Fixed the anchor links and updated the PR.

File: docs/run-move-locally.md (hunk 3 above)
Comment (archjayas) [reply docs/run-move-locally.md]: Fixed the anchor link and updated the PR.

File: docs/run-move-locally.md (hunk 4 above)
Comment (archjayas) [reply docs/run-move-locally.md]: Added a description for 0.

**Comment by ericnakagawa** [comment]
Thanks for the great work team!

**Pull request merged**